peodd/peodd.py

```diff
@@ -28,20 +28,24 @@
 
 import click
 import tomli
-
 from release_tools.project import Project
-from release_tools.semverup import find_pyproject_file
 from release_tools.repo import RepositoryError
+from release_tools.semverup import find_pyproject_file
 
 VERSION_NUMBER_REGEX = r"\d+(?:\.\d+)+"
 
 
 @click.command()
 @click.option('-o', '--output',
-              default="requirements-dev.txt",
+              default="requirements.txt",
               show_default=True,
               help="Output file for the dependencies")
-def main(output):
+@click.option('-m', '--main',
```

Comment:
Instead of `main` flag, how about `--not-dev`?

Reply:
```python
@click.option('-n', '--not-dev',
              default=False,
              is_flag=True,
              show_default=True,
              help="Output not-dev dependencies")
```

Reply:
I see there are a lot of non-dev references in the readme and tests, `non-dev` also sounds better than `not-dev`. Do you mind changing it?

Also, I think `-n` is not needed. I would keep it simple
```
@click.option('--non-dev',
```